pkg/kn/commands/service/migrate.go

```diff
@@ -167,7 +167,7 @@ func NewServiceMigrateCommand(p *commands.KnParams) *cobra.Command {
 				if err != nil {
 					return err
 				}
-				fmt.Println("Migrate service", color.CyanString(service_s.Name), "Successfully")
+				fmt.Println("Migrate service", service_s.Name, "Successfully")
```

Comment:
Change to “Migrated service”

Reply:
Fixed in https://github.com/knative/client/pull/406/commits/bb3fd28369fc262e8757bce812b525effe74fe3d thanks!